vortex-layout/src/lib.rs

```diff
@@ -20,13 +20,16 @@ pub mod stats;
 mod strategy;
 pub mod vtable;
 mod writer;
+pub mod writers;
 
 /// The layout ID for a flat layout
-pub(crate) const FLAT_LAYOUT_ID: LayoutId = LayoutId(1);
+pub const FLAT_LAYOUT_ID: LayoutId = LayoutId(1);
```

Comment:
I particularly liked `mod forever_constant` in vortex-file with a test that checks these values... just because it makes it blindingly obvious that these constants can NEVER be changed

Reply:
I have a FLUP to get rid of these stable IDs. Stable IDs can be strings, that map to u16s within the context of a file